ios/FluentUI/Tab Bar/TabBarItemTokens.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `TabBarItem`.
+open class TabBarItemTokens: ControlTokens {
```

Comment:
Do we need this class to be open?
There's currently no need for the client apps to subclass it as the TabBarItemView is not public and we have not public property of this type.

It seems more lightweight to have these properties in the TabBarViewTokens as it was before.